src/mount.c

```diff
@@ -35,66 +38,146 @@ pu_create_mount_point(const gchar *name,
     return mount_point;
 }
 
+gboolean
+pu_find_mounted(const gchar *device,
+                GList **mounted,
+                GError **error)
```

Comment:
As this function is only used internally, I think you can mark it as `static`.

And then, because it is an internal function, there is probably no need for `GError` handling and return a `GList` directly on success or `NULL` on failure.

Reply:
I will mark it as `static` and remove the `GError`.
Returning  a `GList` directly has the problem that an empty list will return `NULL`, so an error case and an empty list can not be distinguished.